src/w3c/templates/headers.js

```diff
@@ -5,13 +5,25 @@ import showLink from "./show-link";
 
 export default conf => {
   const html = hyperHTML;
+  const specTitleElem = document.querySelector("h1#title, body>h1") || document.createElement("h1") ;
+  if (specTitleElem.parentElement) { // remove it from the DOM tree, because we are going to relocate it
+    specTitleElem.remove();
+    specTitleElem.innerText.trim();
```

Comment:
It should be `specTitleElem.innerText = specTitleElem.innerText.trim();` to be effective, but why should we do this? Also, I would prefer `textContent` which is a whole lot faster than `innerText`, [at least on some browsers](https://kellegous.com/j/2013/02/27/innertext-vs-textcontent/):

![image](https://user-images.githubusercontent.com/3396686/36708092-ba4569b4-1bb4-11e8-980f-29ab64b3f376.png)

Reply:
We definitely don't want to rewrite the content BTW. See the original bug, as the whole point of this is to retain the custom markup in the `h1`.

Reply:
@saschanaz good suggestion about accessing `.textContent` btw.

Reply:
>We definitely don't want to rewrite the content BTW.

Right, I just had no idea about what the line does.

Reply:
I think @shikhar-scs had overlooked that :) 

@shikhar-scs, be mindful that calling `.trim()` doesn't actually trim the `.textContent` - it returns a trimmed *copy* of whatever is in `.textContent`.

Reply:
Err.. I mean "innerText"... you know what I'm saying :)

Reply:
Yes yes I get your point. Thankyou @marcoscaceres and @saschanaz for this detailed review. Ill get started with the changes soon :sweat_smile: